reports/performance/_sales.qmd

```diff
@@ -257,9 +260,19 @@ map2(tri_sales, names(tri_sales), function(x, title) {
     ) +
     facet_wrap(vars(time_sale_year))
 })
+
```

Comment:
This created a loop to print the plots. It was hard to decipher why exactly $South was being printed, but this does the trick.